TechAndCheck/zenodotus
# 275 insights exports

This PR adds `MediaReview`/`ClaimReview` data export functionality to Zenodotus. It uses `BluePrinters` specs created per #390 and #391 to generate JSON exports and a new gem, `comma`, to generate CSV exports. 

The CSV exports are a bit hairy, because they're flat representations of nested data. Column names in the CSV use use underscores to denote nested attributes. 

For example, data corresponding to the `reviewRating.ratingValue` attribute will be written to a  column named `reviewRating_ratingValue`. 

Closes #275 

# Testing instructions
- `bundle` (install `comma`)
- `rake`
- Log into FCI and try out the CSV and JSON data downloads. Make sure the data in the downloaded files matches the data in our `seeds.rb` file.
